README.md

```diff
@@ -44,6 +55,13 @@ _NOTE:_ Commits and pull requests to FINOS repositories will only be accepted fr
 
 *Need an ICLA? Unsure if you are covered under an existing CCLA? Email [help@finos.org](mailto:help@finos.org)*
 
+## Roadmap
+- Support External message template
```

Comment:
That would be nice to create GH issues (with a short description) for each roadmap item, and link them here.

Reply:
done